1.Exploratory_Data_Analysis/1.umap_analysis_plate2.ipynb

```diff
@@ -43,19 +43,13 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    # df = df.sample(n=1000, random_state=0, weights='fourb_Metadata_Treatment_Dose_Inhibitor_Dose')
</pre><p>May have not seen it, but consider removing this code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/48/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
removed line